sdk/cognitivelanguage/Azure.AI.Language.QuestionAnswering/tests/Infrastructure/QuestionAnsweringProjectsLiveTestBase.cs

```diff
@@ -0,0 +1,79 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.AI.Language.QuestionAnswering.Projects;
+using Azure.Core;
+using Azure.Core.TestFramework;
+
+namespace Azure.AI.Language.QuestionAnswering.Tests
+{
+    public partial class QuestionAnsweringProjectsLiveTestBase : QuestionAnsweringTestBase<QuestionAnsweringProjectsClient>
+    {
+        public QuestionAnsweringProjectsLiveTestBase(bool isAsync, QuestionAnsweringClientOptions.ServiceVersion serviceVersion)
+            : base(isAsync, serviceVersion, null /* RecordedTestMode.Record /* to record */)
+        {
+        }
+
+        protected string CreateTestProjectName()
+        {
+            Recording.DisableIdReuse();
```

Comment:
I'm not familiar with this, but perusing the code, I'm curious why you use it here? Of course we can't create the same IDs at the same time, but using the same ID *per test run* against test resources - as long as they get cleaned up - should be fine. Or is this only a problem because we're currently using static resources and old test resources may not have been cleaned up?

Reply:
Yea I was considering the situation where an old test on a static resource may have not deleted the project it created. (I do call a delete project operation at the end of every test, but if the test fails, that part may not be executed).

I also recall using `Recording.DisableIdReuse()` in Translation when we were creating test blob containers and documents for translation. Although now that I think about it, that case was a bit different since we did not attempt to delete those containers after testing. 

Do you think we should keep it just in case?

Reply:
Talked to @pakrym and it only affects the recordings anyway - not the live tests - so probably worth removing. Sounds like it's not quite as effective now as it was originally designed to be anyway.

If we get conflicts on static live resources, it will be further incentive to get per-run live resources provisioned sooner than later.

Reply:
Oh, I see. Removed